packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -182,66 +181,78 @@ export default function SubmitArticleModal({
   }
 
   return (
-    <ResponsiveModal
-      {...modalProps}
-      onRequestClose={onRequestClose}
-      padding={false}
-      style={{ content: { maxWidth: '35rem', width: '100%' } }}
-    >
-      <header className="flex justify-between items-center px-6 w-full h-14 border-b border-theme-divider-tertiary">
-        <div className="flex flex-row items-center">
-          <h3 className="font-bold typo-title3">{headerCopy}</h3>
+    <form ref={submitFormRef} aria-busy={isValidating} onSubmit={onSubmit}>
```

Comment:
I'm not quite sure I like the form wrapping around the whole modal.
Can we not move it into the body and use form="id" on the button rather?

Reply:
Sure thing, but what is the downside?

Reply:
Done 👍

Reply:
No technical downside more of a structural point of view.
It felt like abusing the wrapping.